ios/swift/AmpliSwiftSampleApp/Shared/AmpliSwiftSampleAppApp.swift

```diff
@@ -46,6 +46,9 @@ struct AmpliSwiftSampleAppApp: App {
         // Identify
         ampli.identify(userId: "ampli-swift-user", properties: IdentifyProperties(optionalArray: ["optional string"], requiredNumber: 22.0))
         
+        // Set group
+        ampli.setGroup(name: "ampli group type", value: "ampli swift group")
```

Comment:
Not sure how non-Swifty it is but feel like we could make these unnamed args since they are required and the interface is unlikely to change. WDYT?

```
ampli.setGroup("ampli group type", "ampli swift group")
```

Reply:
done, updated